web/assets/scss/firm-hub.scss

```diff
@@ -70,3 +70,24 @@ main {
 .govuk-form-group {
   max-width: 40em;
 }
+
+span.rag {
+  font-weight: bold;
+  &.green {
+    color: #28a197 !important;
+  }
+  &.amber {
+    color: #f47738 !important;
+  }
+  &.red {
+    color: #d4351c !important;
+  }
+}
+
+span.secondary {
+  display: block;
+  color: #505a5f;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
```

Comment:
It'd also be worth investigating if this can be done with existing GDS classes (it should be possible but we have added CSS before where it's not the case). Let me know if you want a hand looking into it. It'd probably be a good time to check out the GDS patterns library which has examples of the code and it's effects [here](https://design-system.service.gov.uk/styles/layout/)